client/state/http/test/actions.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { http } from '../actions';
+
+describe( '#rawHttp', () => {
+	it( 'should set request parameters', () => {
+		const url = 'http://yury.com',
+			method = 'POST',
+			headers = [
+				[ 'Content-Type', 'application/json' ]
+			],
+			body = {
+				hello: 'world'
+			},
+			withCredentials = true,
+			onSuccess = { type: 'SUCCESS' },
+			onFailure = { type: 'FAIL' };
```

Comment:
Nitpick, but since we started using `const` we have tended to prefer one `const` per variable/line, which is a break from how we aligned the spacing and used a single `var` in the past.